query/src/sql/exec/pipeline_builder.rs

```diff
@@ -487,12 +487,35 @@ impl PipelineBuilder {
         mut child_pipeline: NewPipeline,
         pipeline: &mut NewPipeline,
     ) -> Result<()> {
+        let predicate = other_conditions.iter().cloned().fold(
```

Comment:
You can use `https://doc.rust-lang.org/std/iter/trait.Iterator.html#method.try_reduce` instead.

Reply:
> You can use https://doc.rust-lang.org/std/iter/trait.Iterator.html#method.try_reduce instead.

It's unstable feature, I don't want to enable it...